.gitignore

```diff
@@ -6,6 +6,9 @@ node_modules
 apify_storage
 crawlee_storage
 storage
+.DS_Store
+
+!package.json
```

Comment:
great catch! would be great to add two more too

```suggestion
!package.json
!package-lock.json
!tsconfig.json
```

Reply:
I completely agree, I'll send PR.